src/plots/gl3d/scene.js

```diff
@@ -559,11 +559,13 @@ proto.destroy = function() {
 // for reset camera button in mode bar
 proto.setCameraToDefault = function setCameraToDefault() {
     // as in Gl3d.layoutAttributes
-    this.glplot.camera.lookAt(
+    var lookAtInput = [
         [1.25, 1.25, 1.25],
         [0, 0, 0],
         [0, 0, 1]
-    );
+    ];
+    this.glplot.camera.lookAt.apply(this, lookAtInput);
+    this.graphDiv.emit('plotly_relayout', lookAtInput);
```

Comment:
We could do even better passing the corresponding [camera attribute](https://github.com/plotly/plotly.js/blob/3180b1254668b8791e20766f77707d5601d69a49/src/plots/gl3d/layout/layout_attributes.js#L41-L66) update object e.g:

```js
this.graphDiv.emit('plotly_relayout', {
  scene: {
    eye: { x: 1.25, y: 1.25, z: 1.25 },
    center: { x: 0, y: 0, z: 0 },
    up: { x: 0, y: 0, z: 1 }
  }
});
```